CMakeLists.txt

```diff
@@ -262,5 +262,5 @@ set(USB_IDS_PATH ${CMAKE_BINARY_DIR}/${USB_IDS_FILE})
 set(USB_IDS_INSTALL_PATH ${CMAKE_INSTALL_DATAROOTDIR}/${USB_IDS_FILE})
 
 if(ENABLE_USB_IDS_DOWNLOAD AND NOT EXISTS "${CMAKE_STAGING_PREFIX}/${USB_IDS_PATH}")
```

Comment:
Doesn't this also need to be `USB_IDS_INSTALL_PATH`? That also seems to be the last use of `USB_IDS_PATH` (unless I'm missing an indirect use), so we should probably remove that variable for good (and, while at it, do the same for `PCI_IDS_PATH`).

Reply:
Ah yes, looks like it (on both counts)